### PR TITLE
Moves Chemistry's welding tank to a wall-mounted version

### PR DESCRIPTION
**Pull request by Crazylemon64**
This gives them more space, and one less thing to worry about accidentally pushing while wandering the lab. It behaves as normal otherwise, so don't go making crank in there!
newchem
It's on the far left wall, above the table where the science goggles spawn, between the two chemists.

My one reservation - it might be hard to spot for new players. Thoughts?
:cl:
rscadd: Changes chemistry's welding tank to a wall-mounted version, to increase the room available.
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
:+1:

File: code/modules/surgery/helpers.dm [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/proc/attempt_initiate_surgery(obj/item/I, mob/living/M, mob/user, var/override )
```
Comment (Fox-McCloud) [inline code/modules/surgery/helpers.dm]: What does this have to do with the PR?

File: code/game/machinery/kitchen/smartfridge.dm [hunk 2]
```diff
@@ -135,6 +143,17 @@
 		return 1
 	return 0
 
+/obj/machinery/smartfridge/secure/chemistry/chemcache
+	name = "\improper Smart Chemistry Chem Cache"
+	desc = "A refrigerated storage unit for holding chemical mixtures for later use."
+	initial_contents = list(
```
Comment (Fox-McCloud) [inline code/game/machinery/kitchen/smartfridge.dm]: This isn't documented anywhere in your PR

**Comment by Aurorablade** [comment]
The prs have crossed ahhhhhhhh

File: code/game/machinery/kitchen/smartfridge.dm (hunk 2 above)
Comment (Crazylemon64) [inline code/game/machinery/kitchen/smartfridge.dm]: Ah, well... I tried to make a replacement for the reagents storage locker that would allow for better reagents management even after the supply had been depleted, but I couldn't manage to get it to work - it wouldn't fill with the items on map spawn. So, I've got the vestigial code for it here, even though I don't actually use it.

File: code/modules/surgery/helpers.dm (hunk 1 above)
Comment (Crazylemon64) [inline code/modules/surgery/helpers.dm]: oooooops

**Comment by Fox-McCloud** [comment]
>The prs have crossed ahhhhhhhh

Don't cross the PRs? =p

**Comment by Gummin** [comment]
almost died reading that. such a good idea, hate lugging those things around. :+1: :100:

**Comment by Kyep** [comment]
Good idea. The position of that welding tank is a frequent source of annoyance for medbay chemists.
